WordPress/Classes/Services/PageLayoutService.swift

```diff
@@ -47,4 +52,16 @@ class PageLayoutService {
         }
         return try? JSONDecoder().decode(GutenbergPageLayouts.self, from: data)
     }
+
+    // Parameter Generation
+    private static func parameters() -> [String: AnyObject] {
```

Comment:
Why are we using a static _function_ here, but a static _let block_ for `supportedBlocks`? Obviously the function works fine, it just made me expect to find some dynamic data in the parameters, so I was curious. Maybe to accommodate adding dynamic data in the future?

Reply:
Good question. I should have tied these two together. In [another PR](https://github.com/wordpress-mobile/WordPress-iOS/pull/14795/files#diff-6b53ef68b71b06814a0a684970b2531bR57) I had the need to pass data in from the requesting caller which would have had more dynamic data. So I made this a `func` just to make that later merge easier.